utils/upload-contract.ts

```diff
@@ -0,0 +1,47 @@
+import { DesmosClient, SigningMode, OfflineSignerAdapter } from "@desmoslabs/desmjs";
+import * as Config from "./config"
+import { program } from "commander"
+import * as fs from "fs";
+
+
+interface ProgramOptions {
+    contract: string
+}
+
+async function main() {
+    program
+        .name("Contract uploader")
+        .requiredOption("--contract [contract]", "Path to the contract to upload");
+    program.parse();
+
+    const options = program.opts<ProgramOptions>();
+    // Checks if the smart contract is present.
+    if (!fs.existsSync(options.contract)) {
+        console.error(`Smart contract ${options.contract} don't exist`);
+        return 1;
```

Comment:
```suggestion
        return;
```
Function error should be -1 in C. In addition, I think we can just fast return nothing in javascript.